felix/dataplane/linux/endpoint_mgr.go

```diff
@@ -669,12 +669,14 @@ func (m *endpointManager) resolveWorkloadEndpoints() {
 	}
 
 	removeActiveWorkload := func(logCxt *log.Entry, oldWorkload *proto.WorkloadEndpoint, id types.WorkloadEndpointID) {
-		// QoS state must be removed before the workload itself is removed
-		if oldWorkload != nil {
-			logCxt.Info("deleting QoS bandwidth state if present")
-			err := m.maybeUpdateQoSBandwidth(oldWorkload, nil)
-			if err != nil {
-				logCxt.WithField("workload", oldWorkload).Infof("error deleting QoS bandwidth state, workload may have been already removed: %v", err)
+		if !m.bpfEnabled {
+			// QoS state must be removed before the workload itself is removed
+			if oldWorkload != nil {
+				logCxt.Info("deleting QoS bandwidth state if present")
```

Comment:
```suggestion
				logCxt.Info("Deleting QoS bandwidth state if present.")
```
Please use sentence case for logs.